docs/Running Migrations.md

```diff
@@ -26,30 +26,32 @@ defmodule MyApp.ReleaseTasks do
     :postgrex,
     :ecto
   ]
-  
-  @myapps [
-    :myapp
-  ]
 
-  @repos [
-    MyApp.Repo
-  ]
+  def myapp do
+    __MODULE__
+    |> :application.get_application
+    |> (fn {:ok, app} -> app end).()
```

Comment:
Wouldn't plain old
```
{:ok, app} = :application.get_application(__MODULE__)
app
```
be actually more reasable?

Reply:
I like to see all things are transformations, and maybe things too far (along the same lines as Dave Thomas, https://vimeo.com/216107561)

That said, I can change it to the simpler code (originally I didn't have `myapp` so the piping did more, but now that it's only the one call... just teasing it out would be better)

Reply:
Change updated.